src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirDataStore.cs

```diff
@@ -180,9 +185,16 @@ public async Task<ResourceWrapper> GetAsync(ResourceKey key, CancellationToken c
                             return null;
                         }
 
+                        long resourceSurrogateId;
+                        int version;
+                        bool isDeleted;
+                        bool isHistory;
+                        bool isRawResourceMetaSet = false;
+                        Stream rawResourceStream;
+
                         var resourceTable = VLatest.Resource;
 
-                        (long resourceSurrogateId, int version, bool isDeleted, bool isHistory, Stream rawResourceStream) = sqlDataReader.ReadRow(
+                        (resourceSurrogateId, version, isDeleted, isHistory, rawResourceStream) = sqlDataReader.ReadRow(
```

Comment:
Minor: we can revert back to the declarations inside of the tuple deconstruction:
``` csharp 
(long resourceSurrogateId, int version, bool isDeleted, bool isHistory, Stream rawResourceStream) =
```